suite-native/graph/src/components/GraphFiatBalance.tsx

```diff
@@ -34,7 +34,7 @@ const Skeleton = () => (
 
 const Balance = ({ selectedPointAtom }: BalanceProps) => {
```

Comment:
Would it be possible to show the `valueLatestTotal` even while the graph is loading if we know the value already? It doesn't make sense to hide the current balance by switching graph time frame. And would be awesome to show it sooner even before the graph is loaded.

Reply:
Yes, but it required me to rework this whole PR again :D